File: InterfazJava/ventanasProyecto/src/clases/Servicio.java
import java.util.*;
import java.util.ArrayList;
import java.util.List;
import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.text.ParseException;
import java.util.Date;
public class Servicio {
    private String nombre;
	private float precioxUnit;
	
    public String getNombre() {
        return nombre;
    }
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
	
	public float getprecioxUnit() {
        return precioxUnit;
    }
    public void setprecioxUnit(float precioxUnit) {
        this.nombre = nombre;
    }
	
    public Servicio(String nombre,float precioxUnit){
        this.nombre=nombre;
		this.precioxUnit=precioxUnit;
    }
}

File: InterfazC#/20132952_v4/Modelo/Servicio.cs
using System;
using System.Collections.Generic;
using System.Text;

public class Servicio
{
    private string _nombre;

    public Servicio(string nombre)
    {
        this._nombre = nombre;
    }

    public string Nombre{
		set{
			this._nombre = value;
		}
		get{
			return _nombre;
		}
	}

    //public string Nombre { get => _nombre; set => _nombre = value; }
}
